fvm/state/spock_state.go

```diff
@@ -0,0 +1,172 @@
+package state
+
+import (
+	"encoding/binary"
+	"fmt"
+
+	"github.com/onflow/flow-go/crypto/hash"
+	"github.com/onflow/flow-go/model/flow"
+)
+
+var (
+	// Note: encoding the operation type as part of the spock hash
+	// prevents operation injection/substitution attacks.
+	getMarker         = []byte("1")
```

Comment:
i'm using []byte since otherwise we would be recreating the same byte slice over and over again.  The exact value doesn't matter. we can use any unique non-empty string